decidim-participatory_processes/app/cells/decidim/participatory_processes/process_m_cell.rb

```diff
@@ -39,6 +40,22 @@ def resource_image_path
         model.hero_image.url
       end
 
+      def step_cta_text
+        if translated_in_current_locale?(model.active_step&.cta_text)
+          translated_attribute(model.active_step.cta_text)
+        else
+          t(model.cta_button_text_key, scope: "layouts.decidim.participatory_processes.participatory_process")
+        end
+      end
+
+      def step_cta_path
+        if model.active_step&.cta_path.present?
+          resource_path + "/" + model.active_step.cta_path
```

Comment:
Hi @armandfardeau 
making some tests, I've seen that if there's an url param, like locale, due to having changed language by dropdown, the resulting step_cta_path looks a bit weird, as follows: 
- /processes/blanditiis-labore?locale=ca/test-path-url
![image](https://user-images.githubusercontent.com/9702463/88385810-607d0b80-cdaf-11ea-8ef8-d2757e70f39c.png)

Reply:
Hi! I'm trying to reproduce the problem you showed, but I can't do it, do you have an idea? Here's what I've tried to do:

```ruby
context "when user switch locale" do
  before do
    visit decidim_participatory_processes.participatory_processes_path
     within_language_menu do
       click_link "Català"
     end
  end

  it "displays the regular cta button" do
    within "#participatory_process_#{participatory_process.id}" do
      expect(page).to have_link("Take action!", href: "/processes/#{participatory_process.slug}/my_path")
    end
  end
end
```

Reply:
Hi @armandfardeau 
The steps to reproduce would be:
1. Change language by dropdown. This adds language locale to url
2. Click on processes menu option
3. Click on process CTA button
4. The url has changed with locale param between "normal" url and custom url  

![cta_record_locale](https://user-images.githubusercontent.com/9702463/88630421-d8e61400-d0b0-11ea-8cd3-e12acca54974.gif)